install/prometheus-agent-mode/install.yml

```diff
@@ -0,0 +1,13 @@
+id: setup-prometheus-agent-mode
+name: Prometheus Agent Mode
+title: Prometheus Agent Mode
+description: Use the Prometheus Agent to scrape Prometheus endpoints in your Kubernetes cluster and send metrics to New Relic using Prometheus remote write.
+
+target:
+  type: integration
+  destination: kubernetes
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com
```

Comment:
The final URL is available (temporary empty doc)
https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/